cmd/helm/install.go

```diff
@@ -209,13 +208,13 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 	return client.Run(chartRequested, vals)
 }
 
-// isChartInstallable validates if a chart can be installed
+// checkIfInstallable validates if a chart can be installed
 //
 // Application chart type is only installable
-func isChartInstallable(ch *chart.Chart) (bool, error) {
+func checkIfInstallable(ch *chart.Chart) error {
```

Comment:
shouldn't the proper signature be `func checkIfInstallable(ch *chart.Chart) bool {`, leaving the implementation of the error message to the caller?

Reply:
`checkIfInstallable` would then be able to convey very little information compared to the current implementation. I have no strong opinion on the matter, but either way, maybe that's a different PR? This one was only intended to simplify the implementation of the function, not redefine its purpose.